### PR TITLE
Juju kube up

### PR DESCRIPTION
**Pull request by mbruzek**
I found some problems with the kube-up script that this pull request addresses. We didn't have the kubectl binary in the correct location. 

Just changing where we download the package from the master, and fixing the kube-down.sh script to remove those files.

### CODE REVIEW

**Comment by chuckbutler** [comment]
@mbruzek Hey thanks for the cleanup in the kube-up script. The auto fetch is a welcome addition

:+1: LGTM

File: cluster/juju/util.sh [hunk 1]
```diff
@@ -30,6 +30,8 @@ source "${JUJU_PATH}/${KUBE_CONFIG_FILE-config-default.sh}"
 source ${JUJU_PATH}/prereqs/ubuntu-juju.sh
 export JUJU_REPOSITORY=${JUJU_PATH}/charms
 KUBE_BUNDLE_PATH=${JUJU_PATH}/bundles/local.yaml
+# The destination directory for the kubectl binary file.
+KUBECTL_DIR=${KUBE_ROOT}/platforms/linux/amd64
```
Comment (eparis) [inline cluster/juju/util.sh]: must be quoted (this whole file does a terrible job quoting bash variable, so lets start making them better)
`KUBECTL_DIR="${KUBE_ROOT}/platforms/linux/amd6"

**Comment by eparis** [comment]
So I don't get it. All you did was move the destination directory and change the KUBECONFIG to point to this new destination dir. What was wrong with the old destination directory?

**Comment by mbruzek** [comment]
@eparis Thanks for the review! The reason for this PR is to get the `kube-up.sh` script to complete successfully. Before this PR the script did not know where kubectl binary was expanded. We found the platforms/... is in the directory search path of `kubectl.sh` so we now expand it there. This change basically makes the juju `kube-up.sh` script work.

Yes the path is hardcoded, becuase the cluster/juju/util.sh script does not yet do architecture detection. We could certainly do that, but the charm only gets the amd64 binary at this point, so detecting architecture did not seem the right thing to do.

Again we need this so the kube-up.sh script completes correctly.

**Comment by eparis** [comment]
So rather than having it find it by random side effect, why didn't you set KUBECTL_PATH ?  Or put it in the $PATH ?

**Comment by eparis** [comment]
to be fair, I'm fine with the change in location AND setting KUBECTL_PATH....

**Comment by eparis** [comment]
ok to test

**Comment by eparis** [comment]
please squash to one commit.

**Comment by eparis** [comment]
I think your squash went badly.....

**Comment by mbruzek** [comment]
@eparis working on it.

**Comment by eparis** [comment]
that looks much happier   :)

**Pull request merged**